TempMail/ContentView.swift

```diff
@@ -9,19 +9,22 @@
 import SwiftUI
 
 struct ContentView: View {
-    
-    var email_addr: String
+//    variable for Email struct
+    @ObservedObject var email = Email()
     
     var body: some View {
         VStack {
+//            title
             Text("Temp Mail")
                 .font(.largeTitle)
                 .padding()
             Spacer()
-            
+//            Generate new email button
```

Comment:
Keep this comment and remove the ones on line 24 and 27, don't need both. Also de-indent to be on same level as rest of code. cmd+a, ctrl+i will auto format